accepted/2021/platform-dependencies/platform-dependencies.md

```diff
@@ -0,0 +1,433 @@
+
+### Dependency-tracking tools
+
+The .NET team wants to provide users with self-service tools so that they can know, a priori, what platform dependencies their specific application has. This allows them to configure the systems that will host their application with confidence. The developers creating these tools need a single source of truth from which to derive .NET's platform dependencies. They author the tools to read from the platform dependency model to get this information.
+
+### Dependency change alerting
+
+A .NET contributor makes a change to the NativeMethods.cs file in their project. They create a PR with their changes. A bot alerts the contributor to the potential need to update .NET's platform dependency model in response to the change in the NativeMethods.cs file. The bot alerts them by adding a special label to the PR and a comment that provides instructions on next steps they should follow.
+
+### Platform EOL
+
+Debian 9 goes EOL and a maintainer of the platform dependency model needs to clean up the model to remove references to Debian 9 since those references are now obsolete. They run a CLI command to update the model automatically:
+
+```console
+dotnet-deps platform remove debian.9
+```
+
+## Requirements
+
+### Goals
+
+* Common schema capable of describing both runtime and toolchain dependencies.
+* Model that limits repetition to make maintenance easier.
+* File format that is machine-readable to allow for automated transformation into other output formats.
+* Ability to describe:
+    * Dependencies for multiple platform types (Linux, Windows, MacOS)
+    * Dependencies from various granularities of .NET components such as shared frameworks and NuGet packages.
+    * Dependencies that exist outside of canonical scenarios, such as opting into specific features like diagnostics.
+* Workflow that .NET contributors can easily follow for maintaining the model.
+
+### Non-Goals
+
+* Any dependency that is included in the deployment of the application is outside the scope of this design. NuGet packages are an example of this. A .NET application's dependency on a .NET package, and any assets contained in those packages (managed, native, or otherwise), is explicitly included in the deployment of the application itself. Therefore, the operating environment is not required to be pre-configured with those specific assets; it'll get them naturally through the deployment of the application. However, a NuGet package may have its own platform dependency (e.g. a Linux package) that is not physically contained in the NuGet package; such a platform dependency would be in scope with this design. The concern addressed here is solely focused on what the operating environment must be pre-configured to contain in order to operate on .NET scenarios.
```

Comment:
I think this needs a bit word tweaking. You are not excluding NuGet package dependencies in the design as they are the core building block of any app and runtime itself ships many native libraries with native dependencies as OOB NuGets.

Reply:
I've reworded this to be more accurate: https://github.com/dotnet/designs/pull/184/commits/1a925925642a9bce2c580498445bf0896a522740